apisix/plugins/proxy-rewrite.lua

```diff
@@ -55,6 +55,11 @@ local schema = {
             type    = "string",
             enum    = {"http", "https"}
         },
+        method = {
+          description = "new method",
+          type    = "string",
+          enum    = {"GET", "POST","PUT","PATCH","DELETE"}
```

Comment:
See https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L431, we should support more HTTP methods.